internal/prompt/prompt.go

```diff
@@ -39,18 +43,33 @@ type Prompt struct {
 	out           output.Outputer
 	analytics     EventDispatcher
 	isInteractive bool
+	isForced      bool
 }
 
 // New creates a new prompter
 func New(isInteractive bool, an EventDispatcher) Prompter {
-	return &Prompt{output.Get(), an, isInteractive}
+	return &Prompt{output.Get(), an, isInteractive, false}
 }
 
 // IsInteractive checks if the prompts can be interactive or should just return default values
 func (p *Prompt) IsInteractive() bool {
 	return p.isInteractive
 }
 
+func (p *Prompt) SetInteractive(interactive bool) {
+	p.isInteractive = interactive
+}
+
+// EnableForce forces confirm prompts to return the force value (which is often different from the
+// non-interactive value).
+func (p *Prompt) EnableForce() {
+	p.isForced = true
+}
+
+func (p *Prompt) IsForceEnabled() bool {
```

Comment:
Can we name this to  be consistent with the Interactive mode?

ie. `SetForced()` and `IsForced()`.